spec/models/category_spec.rb

```diff
@@ -3,4 +3,49 @@
 describe Category, type: :model do
   it { should have_many(:videos) }
   it { should validate_presence_of(:name) }
+
+  describe "#recent_videos" do
+    before :each do
+      @test_category = Category.create(name: "drama")
```

Comment:
Great, I added this, very helpful comment here.  This sped up my tests quite a bit.